grid2op/multi_agent/utils.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+from dataclasses import dataclass
```

Comment:
what is all this file doing ?

Reply:
It gives the possibility to define an agent selector that gives priority to agents.

Reply:
But is it necessary right now ?

Reply:
Yes. It gives the order of agents even in the simple case (c.f. __init__ and step for MultiAgentEnv).